packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1364,3 +2166,49 @@ A Role could either be assigned to the User directly or be assigned to a Group o
 | :------------------ | :----------------------------------------------------------------- |
 | `GROUP`             | Role was assigned to an admin Group of which the User is a member |
 | `USER`              | Role was assigned to the User directly                             |
+
+## Subscription object
+
+### Examples
+
+#### Sample Subscription
+
+```json
+{
+    "notificationType": "IWA_AGENT",
+    "channels": [
+        "email"
+    ],
+    "status": "subscribed",
+    "_links": {
+        "unsubscribe": {
+            "href": "http://rain.okta1.com:1802/api/v1/users/00uuk0UVgUXjkIbPL0g3/subscriptions/IWA_AGENT/unsubscribe",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "http://rain.okta1.com:1802/api/v1/users/00uuk0UVgUXjkIbPL0g3/subscriptions/IWA_AGENT"
+        }
+    }
+}
+```
+
+### Subscription properties
+
+The Subscription object defines several properties:
+
+| Property         | Description                                                  | DataType                                                        | Nullable   | Unique   | Read Only |
+| :--------------- | :----------------------------------------------------------- | :-------------------------------------------------------------- | :--------- | :------- | :-------- |
+| notificationType | Type of subscriptions                                        | [Notification Type](#notification-type)                         | FALSE      | TRUE     | TRUE      |
+| channels         | Array of sources that the user will get notification from    | array of [Channel](#subscription-channel)                       | FALSE      | FALSE    | TRUE      |
```

Comment:
another note probably worth adding - or linking to the existing help doc - is that even if status shows subscribed, the admin may not get a notification because for example they may not have permission on the specific resource. For example, a help desk admin will only received user locked out emails for the users that are in groups to which the admin is scoped.